app/components/publications/overview/reports/generate-report-modal.js

```diff
@@ -240,6 +240,10 @@ export default class GenerateReportModalComponent extends Component {
       'page[size]': 100,
     });
     this.governmentDomains = governmentDomains.toArray().sortBy('label');
+    // everything selected by default
+    for (const governmentDomain of this.governmentDomains) {
+      this.selectedGovernmentDomains.addObject(governmentDomain)
```

Comment:
Just take a copy of `this.governmentDomains` using `.slice(0)`?

Reply:
sounds straightforward, doesn't work though.
The `selectedGovernmentDomains` has 10 items after the `slice(0)`, but nothing is checked.

![image](https://user-images.githubusercontent.com/22245223/175497450-f650ad74-e73a-4973-bf94-8f80b06e8cdf.png)

Checking if I can figure out why

Reply:
right now i'm guessing the helper function `includes` is not aware of the changes to the tracked?
Also, checking and unchecking a box now marks the checkbox, but the list of `selectedGovernmentDomains` remains at 10 in both cases

Reply:
yep